app/templates/cookies/cookie_settings.html

```diff
@@ -0,0 +1,123 @@
+{% extends "_base_page.html" %}
+
+{% block pageTitle %}
+  Cookies - Digital Marketplace
```

Comment:
Please can we replace the dash with 'on': Cookies on Digital Marketplace

Reply:
I may actually change this to reflect the `h1` below - "Change your cookie settings"

Reply:
How about: Change your cookie settings on Digital Marketplace?

Reply:
Sounds good!